src/domain/auth-rebac/services/src/rebac_service_impl.rs

```diff
@@ -103,14 +115,42 @@ impl RebacService for RebacServiceImpl {
         self.rebac_repo
             .delete_entity_property(&account_entity, property_name.into())
             .map(map_delete_entity_property_result)
-            .await
+            .await?;
+
+        let mut writable_state = self.cache_state.state.write().await;
+        if let Some(account_properties_cache) = writable_state
+            .account_properties_cache_map
+            .get_mut(account_id.as_str())
+        {
+            match property_name {
+                AccountPropertyName::IsAdmin => {
+                    account_properties_cache.is_admin = false;
+                }
+            }
+        };
```

Comment:
Since, we currently have only one property, we need to remove `account_properties_cache` pair from the state (read from the map) completely